lib/Alchemy/Phrasea/CLI.php

```diff
@@ -73,9 +73,26 @@ public function __construct($name, $version = null, $environment = self::ENV_PRO
 
         $this->bindRoutes();
 
-        $this['logger'] = $this->extend('logger', function () {
-            return new Console\Logger\ConsoleLogger(new Console\Output\ConsoleOutput(Console\Output\ConsoleOutput::VERBOSITY_DEBUG));
-        });
+        $logger = false;
+
+        if ($this['configuration.store']->isSetup()){
+
+            $config = $this['configuration.store']->getConfig();
+
+            if ((isset($config['console_logger_enabled_environments']) && in_array($environment, $config['console_logger_enabled_environments'])) || $environment == self::ENV_DEV){
```

Comment:
testing env_dev here is useless

Reply:
As I understood the request, console logger should be activated either when in dev environment or in environment set in some configuration file. Should just configuration be checked? So if user set just 'prod', logger will not be activated when in dev?

Reply:
I thought that logger should always be activated if in dev

Reply:
yes, in dev logger is alway on. Since it's tested on line 87 anyway, we don't need a "or" on line 82.

Reply:
Sorry, since I update it few times I forgot, you are completely right - will remove || $environment == self::ENV_DEV clause